src/main/kotlin/no/nav/helsearbeidsgiver/forespoersel/ForespoerselRouting.kt

```diff
@@ -70,19 +87,26 @@ private fun Route.forespoersler(forespoerselService: ForespoerselService) {
 }
 
 private fun Route.filtrerForespoersler(forespoerselService: ForespoerselService) {
-    // Hent forespørsler for tilhørende systembrukers orgnr, filtrer basert på request.
+    // Hent forespørsler for orgnr, filtrer basert på request.
```

Comment:
Vi kan egentlig fjerne "for orgnr" her, eller bare skrive noe ala "Filtrer forespørsler basert på request"